src/js/entities-service/entities/flows.js

```diff
@@ -20,6 +20,9 @@ const _Model = BaseModel.extend({
     DetailsChanged({ attributes }) {
       this.set(attributes);
     },
+    ResourceDeleted() {
+      this.destroy({ isDeleted: true });
+    },
```

Comment:
Like we currently do for actions, flows will be removed from the list when we get a `ResourceDeleted` ws notification.

Reply:
When a flow `ResourceDeleted` happens on the patient flow page, we'll also get `ResourceDeleted` notifications for each of the flow's child actions. 

So in that scenario, each action in the list is removed showing `No Actions` and the flow remains visible to the user. And any changes made to that flow will be reflected in the UI but the api request will fail silently.